code/datums/emotes.dm

```diff
@@ -99,6 +99,8 @@
 		TIMER_COOLDOWN_START(user, type, audio_cooldown)
 		TIMER_COOLDOWN_START(user, COOLDOWN_MOB_AUDIO, 20 SECONDS)
 		playsound(user, tmp_sound, volume, vary)
+		if(HAS_TRAIT(user, TRAIT_EMOTE_CD_EXEMPT))
```

Comment:
just have it check for the cooldown exempt trait when assigning the cooldown in the first place